README.md

```diff
@@ -47,4 +49,4 @@ If you forget to include submodules when cloning, add them with `git submodule u
 ### Artifacts
 - Build produces UPM and Unity asset packages .
 - Unity asset package is available under [releases tab](https://github.com/microsoft/spatialaudio-unity/releases)
-- UPM Package is available via [Microsoft Mixed Reality scoped registry](https://microsoft.github.io/MixedRealityToolkit-Unity/version/releases/2.5.0/Documentation/usingupm.html)
\ No newline at end of file
+- Unity Packages are also available on a UPM feed via [Microsoft Mixed Reality Feature Tool](https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/configuration/usingupm?view=mrtkunity-2021-05)
```

Comment:
Previous line says "Unity asset package" and this line says "Unity Packages".  Should use the same phrase/casing to reference the package.